backend/apps/chats/serializers.py

```
@@ -0,0 +1,258 @@
+
+from apps.membership.models import Membership
+
+from .models import ChatGroup, ChatGroupMember, ChatGroupLabel
+
+User = get_user_model()
+
+
+class ChatGroupLabelSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ChatGroupLabel
+        fields = ["id", "name", "created_at"]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)